pkg/controller/orchestrator/orchestrator_controller_test.go

```diff
@@ -137,27 +150,41 @@ var _ = Describe("Orchestrator controller", func() {
 			mgr, err := manager.New(cfg, manager.Options{})
 			Expect(err).NotTo(HaveOccurred())
 			c = mgr.GetClient()
-			recFn, requests = SetupTestReconcile(newReconciler(mgr))
+			recFn, requests = SetupTestReconcile(newReconciler(mgr, orcClient))
 			Expect(add(mgr, recFn)).To(Succeed())
 			stop = StartTestManager(mgr)
 
 			// wait a second for a request
 			Consistently(requests, reconcileTime).ShouldNot(Receive(Equal(expectedRequest)))
-			Eventually(requests).Should(Receive(Equal(expectedRequest)))
+			Eventually(requests, afterReconcileTime).Should(Receive(Equal(expectedRequest)))
```

Comment:
It think it's more obvious to write `2*reconcileTime` than `afterReconcileTime`

Reply:
`afterReconcileTime` is `200 * time.Milisecond`